modules/glslang/SCsub

```diff
@@ -13,46 +13,47 @@ if env["builtin_glslang"]:
     thirdparty_dir = "#thirdparty/glslang/"
     thirdparty_sources = [
         "glslang/CInterface/glslang_c_interface.cpp",
-        "glslang/MachineIndependent/RemoveTree.cpp",
-        "glslang/MachineIndependent/ParseHelper.cpp",
-        "glslang/MachineIndependent/iomapper.cpp",
-        "glslang/MachineIndependent/propagateNoContraction.cpp",
```

Comment:
This is just alphabetical reordering as I didn't see a specific logic to the previous order (doesn't seem to match upstream `CMakeLists.txt` either which is mostly alphabetical too).

There's one new file added though: `glslang/MachineIndependent/SpirvIntrinsics.cpp`.